app/ui/src/app/customizations/tech-extensions/import/tech-extension-import.component.ts

```diff
@@ -70,8 +70,8 @@ export class TechExtensionImportComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.extensionId = this.route.snapshot.paramMap.get('id');
-    this.extensionName = this.route.snapshot.paramMap.get('name');
+    this.extensionId = this.route.snapshot.paramMap.get('extensionId');
+    this.extensionName = this.route.snapshot.queryParamMap.get('name');
```

Comment:
I have a suspicion that this change will break `getUploadUrl`. Shouldn't there be both `id` and `extensionId` values?
https://github.com/syndesisio/syndesis/blob/a3b6962c89cb7c7b96597e87c5c52140591eb5fe/app/ui/src/app/customizations/tech-extensions/import/tech-extension-import.component.ts#L79

Reply:
I think that too as `extension.id` is what it is used for back-end operations whereas `extension.extensionId` is what UI should be shown to the end user (yeah, confusing)

Reply:
Testing with this change produces the following error when `Update` is performed on existing Extension.
```json
{
  "errorCode": 400,
  "userMsg": "An error has occurred while trying to process the technical extension. Please, check the input file.",
  "developerMsg": "An error has occurred while trying to process the technical extension. Please, check the input file. Invalid key. Cannot contain ., %, $, #, [, ], /, or ASCII control characters 0-31 or 127. Key: :com.github.lburgazzoli:syndesis-extensions"
}
```